### DISCUSSION TITLE
Any raspberry examples provided?

### DISCUSSION THREAD

**Discussion by dsantiago** [Megvii-BaseDetection/YOLOX#954] [General]
Hi guys, is there any raspberry tutorials around or should i just follow OpenVino approach?

**Comment by FateScript** [comment]
Sorry that we never try on raspberry device. You are welcomed to have a try and help to contribute :)

**Reply by dsantiago** [reply]
Ohh ok, i will try it soon then.

**Reply by dsantiago** [reply]
Could run it with some tweaks (using the new Raspbian Bullseye OS for 64 bits), the help comes from this link: https://qengineering.eu/install-pytorch-on-raspberry-pi-4.html  i could run but inference was > 2 sec.... Now trying to figure out how to dry it more without going to C++(ncnn)

**Comment by HimanshuMoliya** [comment]
For raspberry PI you need to use it with tflite : https://github.com/NobuoTsukamoto/tflite-cv-example/tree/master/yolox/python

**Reply by dsantiago** [reply]
Thanks for sharing, what surprised me was that i could get a good inference results with a converted ONNX model and ofc, the onnx library on Raspberry. What gave me a little challenge was installing the 64 bit Raspbian OS and build torch, torch vision, etc on it..
